packages/common/src/ContractInteractor.ts

```diff
@@ -535,24 +536,26 @@ export class ContractInteractor {
       this.logger.debug('relayCall res=' + res)
 
       // @ts-ignore
-      const decoded = abi.decodeParameters(['bool', 'bytes'], res)
+      const decoded = abi.decodeParameters(['bool', 'uint256', 'bytes'], res)
```

Comment:
technically, an enum is `uint8`...

Reply:
In the decoded structure it will not matter.